### PR TITLE
adding sidebar for community docs

### PR DESCRIPTION
**Pull request by Rumyra**
#### Summary

Adding mdn sidebar to the community doc pages, and updating a few links

### CODE REVIEW

File: files/en-us/mdn/community/contributing/getting_started/index.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ tags:
   - governance
 ---
```
Comment (teoli2003) [inline files/en-us/mdn/community/contributing/getting_started/index.md:10]: No space at top of the page 
```suggestion
```

File: files/en-us/mdn/community/contributing/index.md [hunk 2]
```diff
@@ -8,7 +8,9 @@ tags:
   - governance
 ---
```
Comment (teoli2003) [inline files/en-us/mdn/community/contributing/index.md:10]:
```suggestion
```

File: files/en-us/mdn/community/contributing/our_repositories/index.md [hunk 3]
```diff
@@ -8,6 +8,8 @@ tags:
   - governance
 ---
```
Comment (teoli2003) [inline files/en-us/mdn/community/contributing/our_repositories/index.md:10]:
```suggestion
```

File: files/en-us/mdn/community/contributing/security_vulnerability_response/index.md [hunk 4]
```diff
@@ -8,6 +8,8 @@ tags:
   - governance
 ---
```
Comment (teoli2003) [inline files/en-us/mdn/community/contributing/security_vulnerability_response/index.md:10]:
```suggestion
```

File: files/en-us/mdn/community/contributing/translated_content/index.md [hunk 5]
```diff
@@ -8,6 +8,8 @@ tags:
   - governance
 ---
```
Comment (teoli2003) [inline files/en-us/mdn/community/contributing/translated_content/index.md:10]:
```suggestion
```

File: files/en-us/mdn/community/discussions/index.md [hunk 6]
```diff
@@ -8,6 +8,8 @@ tags:
   - governance
 ---
```
Comment (teoli2003) [inline files/en-us/mdn/community/discussions/index.md:10]:
```suggestion
```

File: files/en-us/mdn/community/issues/content_suggestions_feature_proposals/index.md [hunk 7]
```diff
@@ -8,6 +8,8 @@ tags:
   - governance
 ---
```
Comment (teoli2003) [inline files/en-us/mdn/community/issues/content_suggestions_feature_proposals/index.md:10]:
```suggestion
```

File: files/en-us/mdn/community/issues/index.md [hunk 8]
```diff
@@ -8,7 +8,9 @@ tags:
   - governance
 ---
```
Comment (teoli2003) [inline files/en-us/mdn/community/issues/index.md:10]:
```suggestion
```

File: files/en-us/mdn/community/learn_forum/index.md [hunk 9]
```diff
@@ -8,6 +8,8 @@ tags:
   - governance
 ---
```
Comment (teoli2003) [inline files/en-us/mdn/community/learn_forum/index.md:10]:
```suggestion
```

File: files/en-us/mdn/community/mdn_content/index.md [hunk 10]
```diff
@@ -9,6 +9,8 @@ tags:
   - mdn-content
 ---
```
Comment (teoli2003) [inline files/en-us/mdn/community/mdn_content/index.md:11]:
```suggestion
```

File: files/en-us/mdn/community/mdn_content/issues/index.md [hunk 11]
```diff
@@ -9,4 +9,6 @@ tags:
   - mdn-content
 ---
```
Comment (teoli2003) [inline files/en-us/mdn/community/mdn_content/issues/index.md:11]:
```suggestion
```

File: files/en-us/mdn/community/mdn_content/pull_requests/index.md [hunk 12]
```diff
@@ -9,6 +9,8 @@ tags:
   - mdn-content
 ---
```
Comment (teoli2003) [inline files/en-us/mdn/community/mdn_content/pull_requests/index.md:11]:
```suggestion
```

File: files/en-us/mdn/community/mdn_content/issues/index.md [hunk 13]
```diff
@@ -9,4 +9,6 @@ tags:
...
 
+{{MDNSidebar}}
+
 TBD
```
Comment (teoli2003) [inline files/en-us/mdn/community/mdn_content/issues/index.md:14]: We should avoid placeholders. Most of the time, we forget about them forever.

File: files/en-us/mdn/community/open_source_etiquette/index.md [hunk 14]
```diff
@@ -9,6 +9,8 @@ tags:
   - open-source
 ---
```
Comment (teoli2003) [inline files/en-us/mdn/community/open_source_etiquette/index.md:11]:
```suggestion
```

File: files/en-us/mdn/community/pull_requests/index.md [hunk 15]
```diff
@@ -8,6 +8,8 @@ tags:
   - governance
 ---
```
Comment (teoli2003) [inline files/en-us/mdn/community/pull_requests/index.md:10]:
```suggestion
```

File: files/en-us/mdn/community/pull_requests/index.md [hunk 16]
```diff
@@ -8,6 +8,8 @@ tags:
...
 
+{{MDNSidebar}}
+
 ## Guidelines for after submitting a pull request
```
Comment (teoli2003) [inline files/en-us/mdn/community/pull_requests/index.md:13]: We should have a paragraph giving the context before the `h2` header here.

File: files/en-us/mdn/community/users_teams/index.md [hunk 17]
```diff
@@ -8,6 +8,8 @@ tags:
   - governance
 ---
```
Comment (teoli2003) [inline files/en-us/mdn/community/users_teams/index.md:10]:
```suggestion
```

File: files/en-us/mdn/writing_guidelines/index.md [hunk 18]
```diff
@@ -9,7 +9,7 @@ tags:
```
Comment (teoli2003) [inline files/en-us/mdn/writing_guidelines/index.md:9]:
```suggestion
```

File: files/en-us/mdn/community/contributing/security_vulnerability_response/index.md [hunk 19]
```diff
@@ -8,6 +8,8 @@ tags:
...
 
+{{MDNSidebar}}
+
 ## A little history
```
Comment (teoli2003) [inline files/en-us/mdn/community/contributing/security_vulnerability_response/index.md:13]: We should give some context before this `h2`, or we should get rid of the header here.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Comment by teoli2003** [comment]
We have weird broken links appearing in flaws…

Capture d’écran 2022-08-03 à 16 21 42

File: files/en-us/mdn/community/index.md [hunk 20]
```diff
@@ -7,6 +7,7 @@ tags:
   - community-guidelines
   - governance
 ---
+{{MDNSidebar}}
 
 👋 Welcome to MDN Web Docs on GitHub! MDN Web Docs is an open-source, collaborative project that documents web platform technologies, including [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML), [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS), [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript), and [Web APIs](https://developer.mozilla.org/en-US/docs/Web/API). We also provide extensive [learning resources](https://developer.mozilla.org/en-US/docs/Learn) for early-stage developers and students.
```
Comment (MrBrain295) [inline files/en-us/mdn/community/index.md:12]:
```suggestion
👋 Welcome to MDN Web Docs on GitHub! MDN Web Docs is an open-source, collaborative project that documents web platform technologies, including [HTML](/en-US/docs/Web/HTML), [CSS](/en-US/docs/Web/CSS), [JavaScript](/en-US/docs/Web/JavaScript), and [Web APIs](/en-US/docs/Web/API). We also provide extensive [learning resources](/en-US/docs/Learn) for early-stage developers and students.
```

**Pull request merged**